go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/gdamore/tcell v1.3.0
 	github.com/hajimehoshi/go-mp3 v0.3.0
-	github.com/hajimehoshi/oto v0.7.1
+	github.com/hajimehoshi/oto/v2 v2.0.0
```

Comment:
What about using the latest version (v2.3.1)

Reply:
Updated to the newest alpha version because there is no stable version containing a certain bugfix.